app/inpututils.cpp

```diff
@@ -277,6 +277,19 @@ void InputUtils::setExtentToFeature( const FeatureLayerPair &pair, InputMapSetti
   mapSettings->setExtent( currentExtent );
 }
 
+QPointF InputUtils::geometryCenterToScreenCoordinates( const QgsGeometry &geom, InputMapSettings *mapSettings )
+{
+  QPointF screenPoint;
+
```

Comment:
Please add check for validity of the `mapSettings` pointer

Reply:
whoops!